PeripheralController.swift

```diff
@@ -1,119 +1,112 @@
 import CoreBluetooth
 
-public struct PeripheralCharacteristicsDataV2: Codable {
-    var mp: String // phone model of peripheral
-    var id: String // tempID
-    var o: String // organisation
-    var v: Int // protocol version
+public struct PeripheralCharacteristicsData: Codable {
+    var modelP: String // phone model of peripheral
+    var msg: String // tempID
+    var org: String
+    var v: Int
 }
 
-public class PeripheralController: NSObject {
+let TRACER_SVC_ID: CBUUID = CBUUID(string: "\(PlistHelper.getvalueFromInfoPlist(withKey: "TRACER_SVC_ID") ?? "B82AB3FC-1595-4F6A-80F0-FE094CC218F9")")
+let TRACER_SVC_CHARACTERISTIC_ID = CBUUID(string: "\(PlistHelper.getvalueFromInfoPlist(withKey: "TRACER_SVC_ID") ?? "B82AB3FC-1595-4F6A-80F0-FE094CC218F9")")
```

Comment:
These IDs get set as custom Build Settings in the Xcode project. They are the same values as what they've hardcoded.